### ISSUE TITLE
Deauth on 5GHz and Evil on 2.4GHz

### ISSUE THREAD

**Issue by grizzwa** [FluxionNetwork/fluxion#1004]
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes

#### Summary

I have 2 wifi cards:
1. DualBand 2.4/5Ghz
2. 2.4Ghz

What about a feature which let the user choose to deauth the clients on the 5GHz network and create the Evil Twin on the 2.4GHz.
Because it isnt really important if the Evil is on the 5GHz or on 2.4GHz band. They only see the ESSID and the BSSID on some cases or devices, but most of the time they only see the ESSID when connecting.
Also the range is much better on 2.4GHz.

I think [Airgeddon](https://github.com/v1s1t0r1sh3r3/airgeddon) is doing it that way.

Also because of the regulatory domain rules it is more reliable to make the Evil on the 2.4GHz.
There are much more regulations for specific countrys in the 5GHz Band.

**Comment by usuarionuevor** [comment]
Good idea, I've been thinking about something like this for a while now that every new box comes with both 2,4 and 5ghz
I wonder how to implement, because it would be like attacking 2 different AP at the same time, or maybe deauth manually without fluxion

**Comment by chichke** [comment]
It would make dual band wireless adapter like the AWUS036ACH works 🙏

**Comment by usuarionuevor** [comment]
is the project dead already? :(

**Comment by RadioactivePlays** [comment]
I cant see how airgeddon is doing it. Can you explain?

**Comment by lucasedmilsondesouza-pixel** [comment]
https://github.com/FluxionNetwork/fluxion.git
